make/photon/db/Dockerfile.base

```diff
@@ -1,4 +1,4 @@
-FROM photon:2.0
+FROM photon:3.0
```

Comment:
This is going to lead to a new major version of PostgreSQL, right?  That's going to lead to some pretty _major_ headaches for upgrading existing installs (https://github.com/docker-library/postgres/issues/37).

Reply:
Yes. pg version will be majar upgraded

We could use a hack to avoid to
break amd64 user.

```Dockerfile
FROM photon:2.0 as photon-amd64
FROM photon:3.0 as photon-arm64

FROM photon-${TARGETARCH:-amd64}
```

arm64 user could be safe to use with new pg

Reply:
Sure, but that just punts this problem (and creates the new situation where now different architecture deployments will have platform divergence) -- this is something that's going to have to be dealt with at some point relatively soon, I think. :disappointed: :confused:

Reply:
seems we have to custom build pg 9.x instead of installing the distro version

Reply:
Sure, that would be a better temporary solution, but it does still leave the upgrade hanging over our heads (because eventually 9.x will be EOL and we'll have to solve this).

However, to be clear, I'm not a maintainer here (just a very interested contributor), so don't let me monopolize this too much. :sweat_smile:

(Personally, I think using something other than `photon` would be a good path to consider too.)

Reply:
Yeah. i'm using an external pg for harbor 😂